.github/workflows/publish.yml

```diff
@@ -0,0 +1,63 @@
+# Sample workflow for building and deploying a VitePress site to GitHub Pages
```

Comment:
Can I do this please, just a little pedantic about making sure I understand what its doing. And if I make a mistake and it fucks something up its then my fault and not yours 👍